Content.Server/GameTicking/Rules/GameRuleSystem.Utility.cs

```diff
@@ -126,4 +126,8 @@ protected bool TryGetRandomStation([NotNullWhen(true)] out EntityUid? station, F
         return found;
     }
 
+    protected void ForceEndSelf(EntityUid uid, GameRuleComponent? component = null)
```

Comment:
entity<> (tho a helper for this is a little odd since you only save like what, 5 characters?)

Reply:
its a thing from station event system :trollface:

Reply:
anyway not worth removing ForceEndSelf or changing EndGameRule to use Entity because le breaking changes